plugins/chat/app/services/chat/publisher.rb

```diff
@@ -113,12 +113,12 @@ def self.publish_processed!(chat_message)
       )
     end
 
-    def self.publish_edit!(chat_channel, chat_message)
+    def self.publish_edit!(chat_channel, chat_message, streaming: false)
```

Comment:
Do we need to set streaming here, I'm wondering if updating a message would ever be streamed?

Reply:
I think you are right that we alrzady have this info on the chat message now